### ISSUE TITLE
We need a Safari extension.

### ISSUE THREAD

**Issue by neemasa** [manojVivek/medium-unlimited#37]
It would be a great if you could port this into Safari as other browsers take too much RAM and Battery.

**Comment by ravindersaluja** [comment]
Yes!! Like @neemasa said this extension is much needed for the Safari browser!! Works wonders with Firefox! Thanks :)

**Comment by mytogi** [comment]
upvote for that idea - just took a stab at it on firefox.. love it :)

**Comment by sanarena** [comment]
Safari is going to support WebExtensions API (same as Firefox, Edge and Chrome) in v14 coming with Big Sur, so soon you will be able to load this into Safari.

**Comment by ravindersaluja** [comment]
> Safari is going to support WebExtensions API (same as Firefox, Edge and Chrome) in v14 coming with Big Sur, so soon you will be able to load this into Safari.

@sanarena Yes.

**Comment by tgvdinesh** [comment]
[This](https://developer.apple.com/documentation/safariservices/safari_web_extensions) ain't the Safari WebExtensions API?

**Comment by bhavul** [comment]
Hello, I tried creating a Safari extension for this for new Safari 14 and I posted an update here:  
https://github.com/manojVivek/medium-unlimited/issues/7#issuecomment-731847461
